app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -128,6 +127,7 @@
     public static AndroidFormController mFormController;
 
     private boolean mIncompleteEnabled = true;
+    private boolean instanceIsReadOnly = false;
```

Comment:
@amstone326 This needs to be added to the list of serialized case properties save and recovered when the activity cycle restarts or the state won't persist.